ui-tests/cypress/integration/pages-management.create.spec.js

```diff
@@ -0,0 +1,138 @@
+import HomePage from "../support/pageObjects/HomePage";
+import { generateRandomId } from '../support/utils';
+
+let currentPage;
+
+const openManagementPage = () => {
+    cy.visit('/');
+    let currentPage = new HomePage();
+    currentPage = currentPage.getMenu().getPages().open();
+    return currentPage.openManagement();
+}
+
+const OOTB_PAGE_TEMPLATES = ['1-2-column', '1-2x2-1-column', '1-2x4-1-column', '1-column', 'content-page', 'home', 'single_frame_page']
+
+describe('Pages Management - Create', () => {
+    beforeEach(() => {
+        cy.kcLogin("admin").as("tokens");
+    });
+
+    afterEach(() => {
+        cy.kcLogout();
+    });
+
+  it('Create a page for every OOTB page-template', () => {
+    currentPage = openManagementPage();
+
+    OOTB_PAGE_TEMPLATES.forEach(template => {
+        const code = generateRandomId();
+        const friendlyCode = generateRandomId();
+
+        // go to add page
+        currentPage = currentPage.getContent().clickAddButton();
+    
+        currentPage.getContent()
+            .fillRequiredData(`${template} - test en`, `${template} - test it`, code, 0, template);
+        
+        currentPage.getContent().fillSeoData('SEO description', 'keyword', friendlyCode);
+
+        currentPage = currentPage.getContent().clickSaveButton();
+    
+        cy.wait(2000);
+        // page should be created successfully
+        currentPage.getContent().getRowByPageName(`${template} - test en`).should('be.visible');
+    
+        // delete recently added page
+        cy.pagesController().then(controller => controller.deletePage(code));
+        cy.wait(2000);
```

Comment:
What is this wait for?

Reply:
I wanted to add some buffer between each iteration of this loop, for some reason the cypress interface was crashing without reason, and that buffer seemed to help it. I'll remove and see if it still works fine